tests/test_project_structure.py

```diff
@@ -251,3 +250,20 @@ def find_resource_files(department="*", resource_type="*", service="*"):
         # Exclude __init__.py and pycache
         resource_files = (f for f in resource_files if not f.endswith("__init__.py"))
         return resource_files
+
+
+class TestOperatorsHooks(unittest.TestCase):
+    def test_some_suffix_is_illegal(self):
```

Comment:
```suggestion
    def test_no_illegal_suffixes(self):
          illegal_suffixes = ["_operator.py", "_hook.py", "_sensor.py"]
```
WDYT? Will it make the test easier to understand?